tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_node.h

```diff
@@ -90,7 +95,12 @@ const std::string SAVE_MAP_SERVICE_NAME = "save_map";
 const std::string CONNECT_SERVICE_NAME = "connect";
 
 const std::string DATASETS_PATH = "/sdcard/tango_ros_streamer/datasets/";
-const double TANGO_3D_RECONSTRUCTION_DEFAULT_RESOLUTION = 0.05; // meter
+const double TANGO_3DR_DEFAULT_RESOLUTION = 0.05; // meter
+const bool TANGO_3DR_DEFAULT_USE_SPACE_CLEARING = false;
+const int32_t TANGO_3DR_DEFAULT_MIN_NUM_VERTICES = 1;
```

Comment:
It's the default, we should only differ if we have a good reason.
From the Tango developers site: 

> Minimum number of vertices that every connected component in the mesh should consist of. Defaults to 1. Smaller components (typically noise) will be deleted.